diagnostic.md

```diff
@@ -6,21 +6,25 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    The Router, parses the url and directs the user/client/browser to the
+    appropriate route.
+
+    The Route sends and recieves data from the server.
+
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    ember generate route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    {{#link-to campus/boston}}{{/link-to}}
```

Comment:
There's a gotcha on link-to! 
{{#link-to 'campus.boston'}}Boston{{/link-to}}

0/1